src/Main.cc

```diff
@@ -285,9 +310,8 @@ int main(int argc, char* argv[])
         = std::make_unique<ord::Design>(the_tech_and_design.tech.get());
     ord::OpenRoad::setOpenRoad(the_tech_and_design.design->getOpenRoad());
     ord::initOpenRoad(interp, log_filename, metrics_filename);
-    if (!findCmdLineFlag(cmd_argc, cmd_argv, "-no_splash")) {
-      showSplash();
-    }
+
+    launched(cmd_argc, cmd_argv);
```

Comment:
`launched()` does not sounds like verb a function typically would be named.

Reply:
fixed